docs/source/reference/visualization/index.rst

```diff
@@ -27,6 +27,7 @@ The :mod:`~optuna.visualization` module provides utility functions for plotting
    optuna.visualization.plot_param_importances
    optuna.visualization.plot_pareto_front
    optuna.visualization.plot_slice
+   optuna.visualization.plot_timeline
```

Comment:
Ah, the document build failed. We need to update `optuna/visualization/__init__.py` to expose `plot_timeline` there.

```
IPython could not be loaded!
WARNING: [autosummary] failed to import optuna.visualization.plot_timeline.
Possible hints:
* ModuleNotFoundError: No module named 'optuna.visualization.plot_timeline'
* AttributeError: module 'optuna.visualization' has no attribute 'plot_timeline'
* ImportError: 
```

Reply:
I confirmed that the reference was rendered as expected.